Assets/Scripts/Models/Temperature.cs

```diff
@@ -1,6 +1,6 @@
 ﻿#region License
 // ====================================================
-// Project Porcupine Copyright(C) 2016 Team Porcupine
+// Project Porcupine Copyright(c) 2016 Team Porcupine
```

Comment:
I don't know what that was about. I got the conflict after SCS and just rolled with it. It was nothing to do with my work